src/Schema/Tables/PatientAllergy.sql

```diff
@@ -0,0 +1,11 @@
+﻿CREATE TABLE [dbo].[PatientAllergy]
+(
+	[PatientAllergyId] INT NOT NULL PRIMARY KEY IDENTITY, 
+    [PatientId] INT NOT NULL, 
+    [AllergyId] INT NOT NULL,
+	[Duration] NVARCHAR(25) NULL,
+    [Notes] TEXT NULL, 
+    [IsDeleted] BIT NOT NULL, 
+    CONSTRAINT [FK_PatientAllergy_To_Allergies] FOREIGN KEY (AllergyId) REFERENCES Allergies(AllergyId), 
+    CONSTRAINT [FK_PatientAllergy_To_Patient] FOREIGN KEY (PatientId) REFERENCES PatientInfo(PatientId)
```

Comment:
Why do you write PatientInfo(PatientId)? Do you know that Alexandr will use PatientId? Or may be he will just Id like in our schema?